package-lock.json

```diff
@@ -0,0 +1,19923 @@
+{
```

Comment:
I don't think this file is supposed to be committed? I could be wrong @PreetPatel

Reply:
yup this shouldn't be here.. never use NPM :)))

Reply:
Yeah, I'm not sure if I should add package.json or package-lock.json for react-to-print.